core-plugins/widgets/FTP-batchsource.json

```diff
@@ -27,27 +32,22 @@
           "name": "fileSystemProperties"
         },
         {
-          "widget-type": "select",
+          "widget-type": "radio-group",
           "label": "Ignore Non-Existing Folders",
```

Comment:
been labeling this as Allow Empty Input in other plugins, let's use that here too